### DISCUSSION TITLE
Authenticator - Login failure event

### DISCUSSION THREAD

**Discussion by ldecoker** [symfony/symfony#42489] [Q&A] [answered]
Hello!
On symfony 5.3, with the new authenticator system, when an authentication failed, if I understood it well, a LoginFailureEvent is triggered. 
I have so created an event subscriber on AuthenticationEvents::AUTHENTICATION_FAILURE but when testing it, nothing happens (log the failure event).
Do I listen on the correct event? 

Thx!

**Comment by ahundiak** [comment, marked as answer]
Symfony\Component\Security\Http\Event\LoginFailureEvent

**Reply by ldecoker** [reply]
Thx! It works.
What if the user does not exist?

**Reply by ahundiak** [reply]
I have not tried it with the new system yet but I would imagine that [Hide User Not Found](https://symfony.com/doc/current/reference/configuration/security.html#hide-user-not-found) is still relevant.
